pkg/flags/flags.go

```diff
@@ -277,6 +293,7 @@ func ParseArgs(args []string) (*EnvFlags, error) {
 		dryRun:                  dryRun,
 		failFast:                failFast,
 		disableGracefulTeardown: disableGracefulTeardown,
+		context:                 context,
```

Comment:
@vladimirvivien I know you dissed about keep this consistent with the kubectl in terms of args. Which Is perfectly fine. But I think we should use the internal variables/attributes in such a way that it won't conflict with the `context` package. That can lead to a lot of confusion. Do you think it will be ok to at least name the internal attributes/variables as `kubeContext` but retain the flag argument as `--context` ?

Reply:
Not @vladimirvivien but I think you have a point and I can change that. I didn't think about the `context` package and the shadowing at all when writing this.

Reply:
@maruina @harshanarayana Yes, I thought about `context` causing confusion with context package. So, what I am suggesting is just the name of the flag being called `context`. However internally, the variable name should be kubeContext (everywhere it's used).  Hope that makes sense.

Reply:
I think after that, this will be ready to go and we can have a release soon.

Reply:
@vladimirvivien  @maruina  Works for me.